js/repl/Utils.js

```diff
@@ -0,0 +1,25 @@
+const sizes = ["Bytes", "kB", "MB", "GB", "TB", "PB", "EB"];
```

Comment:
It would be great to specify where was this code inspired from: https://github.com/davglass/prettysize
Something like:
`Forked from https://github.com/davglass/prettysize`